content/en-us/reference/engine/classes/TweenBase.yaml

```diff
@@ -76,6 +76,11 @@ methods:
 
       If you want to reset the progress variables of the tween, use
       `Class.TweenBase:Cancel()`.
+
+      You can only pause tweens that are in the `Class.TweenBase.PlaybackState|PlaybackState`
+      of `Enum. PlaybackState.Playing`; tweens in other states won't pause.
+      To keep track of the current `PlaybackState`, you can listen to the `PlaybackState`
+      property change events.
```

Comment:
Is there a specific reason this usage is being noted here? Is it necessary to listen to a property change on tweens to accomplish some other specific intention? If so, we should provide an example of that to frame this practice in context.

Reply:
The previous sentence is the context and the example of this is already in the previous Code sample.

In other words, they need to know the `PlaybackState` in order to know if they can use the `Pause()`.

Reply:
For more context, please check out the DevForum ticket linked in the description.